cmd/checkmetrics/README.md

```diff
@@ -1,5 +1,23 @@
 # checkmetrics
 
+* [checkmetrics](#checkmetrics)
```

Comment:
slightly confused - should this and the next patch maybe folded into one?

Reply:
Ah - that link can in fact be zapped. But I'm not quite clear on what you mean - the next commit is for `checkcommits` ?There are a number of ways these changes could be ordered but I've tried to patch-up the existing files, then finally add in missing links.

Reply:
doh, I mis-read and muddle up `checkcommits` and `checkmetrics` in my head all the time - heh, and I tried to stare quite hard at this one - mea culpa - ignore me!